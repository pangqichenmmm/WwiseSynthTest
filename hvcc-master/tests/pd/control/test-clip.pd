#N canvas 630 159 312 269 10;
#X obj 43 43 loadbang;
#X obj 43 209 print;
#X obj 112 43 bng 15 250 50 0 empty empty empty 17 7 0 10 -262144 -1
-1;
#X obj 43 81 t b b b b b;
#X msg 122 125 0.5;
#X msg 160 126 1;
#X msg 198 126 2;
#X msg 43 125 -1;
#X msg 78 125 -0.1;
#X obj 43 174 clip 0 1;
#X obj 134 177 clip;
#X connect 0 0 3 0;
#X connect 2 0 3 0;
#X connect 3 0 7 0;
#X connect 3 1 8 0;
#X connect 3 2 4 0;
#X connect 3 3 5 0;
#X connect 3 4 6 0;
#X connect 4 0 9 0;
#X connect 5 0 9 0;
#X connect 6 0 9 0;
#X connect 7 0 9 0;
#X connect 8 0 9 0;
#X connect 9 0 1 0;
