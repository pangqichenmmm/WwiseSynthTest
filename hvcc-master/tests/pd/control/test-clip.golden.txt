[@ 0.000] print: 1
[@ 0.000] print: 1
[@ 0.000] print: 0.5
[@ 0.000] print: 0
[@ 0.000] print: 0
